tests/unit/test_download.py

```diff
@@ -141,8 +146,11 @@ def test_path_to_url_win():
 
 @pytest.mark.skipif("sys.platform != 'win32'")
 def test_url_to_path_win():
-    assert url_to_path('file:///c:/tmp/file') == 'C:\\tmp\\file'
+    assert url_to_path('file:tmp') == 'tmp'
+    assert url_to_path('file:///c:/tmp/file') == r'C:\tmp\file'
     assert url_to_path('file://unc/as/path') == r'\\unc\as\path'
+    assert url_to_path('file:c:/path/to/file') == r'C:\path\to\file'
+    assert url_to_path('file://localhost/c:/tmp/file') == r'C:\tmp\file'
```

Comment:
See my other ordering comment.

Reply:
I also think it would be nice if the test cases that are supposed to run on all platforms are broken out into a third test. That way there will be less repetition of test cases, and it's easier to see which cases are the same on both platforms.